src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/components/SceneFunctionDetailsPanel/components/GitHubContextProvider/infrastructure/GitSessionCookieManager.ts

```diff
@@ -25,50 +27,67 @@ class InternalGitSessionCookieManager implements GitSessionCookieManager {
   }
 
   setCookie(cookie: string): void {
-    if (!cookie.startsWith(`${GITHUB_SESSION_COOKIE_NAME}=`)) {
+    if (
+      !cookie.startsWith(`${LEGACY_GITHUB_SESSION_COOKIE_NAME}=`) &&
+      !cookie.startsWith(`${GITHUB_SESSION_COOKIE_NAME}=`)
+    ) {
+      // If incoming cookie is not named, we use the new cookie name
       cookie = `${GITHUB_SESSION_COOKIE_NAME}=${cookie}`;
     }
 
-    const rawCookie = InternalGitSessionCookieManager.getCookieFromJar(cookie, GITHUB_SESSION_COOKIE_NAME);
+    const rawCookie = InternalGitSessionCookieManager.getCookieFromJar(cookie, ACCEPTED_GITHUB_SESSION_COOKIE_NAMES);
     if (rawCookie === undefined) {
       // If we can't parse the key-value pair out of [cookie], let's return now
       // to avoid corrupting the state of the manager or browser cookie.
       return;
     }
 
+    if (rawCookie.key === GITHUB_SESSION_COOKIE_NAME) {
+      // If new name is present, let's delete old one
+      this.deleteLegacyCookie();
+    }
+
     this.rawCookie = rawCookie;
     this.sessionCookie = GitSessionCookie.decode(rawCookie.value);
     document.cookie = `${cookie}; path=/`;
   }
 
   deleteCookie(): void {
-    document.cookie = `${GITHUB_SESSION_COOKIE_NAME}=; expires=Thu, 01 Jan 1970 00:00:00 UTC;`;
+    document.cookie = `${GITHUB_SESSION_COOKIE_NAME}=; Path=/; expires=Thu, 01 Jan 1970 00:00:00 UTC;`;
+    this.deleteLegacyCookie();
     this.rawCookie = undefined;
     this.sessionCookie = undefined;
   }
 
+  private deleteLegacyCookie(): void {
+    document.cookie = `${LEGACY_GITHUB_SESSION_COOKIE_NAME}=; Path=/; expires=Thu, 01 Jan 1970 00:00:00 UTC;`;
```

Comment:
In Chrome, I had to set `Path=/;` to delete the cookie. Before using this, the cookie was still visible in the browser